bundles/org.openhab.core.compat1x/META-INF/MANIFEST.MF

```diff
@@ -1,105 +1,110 @@
- org.codehaus.jackson.map.ext,
- org.codehaus.jackson.map.introspect,
- org.codehaus.jackson.map.jsontype,
- org.codehaus.jackson.map.jsontype.impl,
- org.codehaus.jackson.map.module,
- org.codehaus.jackson.map.ser,
- org.codehaus.jackson.map.ser.impl,
- org.codehaus.jackson.map.ser.std,
- org.codehaus.jackson.map.type,
- org.codehaus.jackson.map.util,
- org.codehaus.jackson.node,
- org.codehaus.jackson.schema,
- org.codehaus.jackson.sym,
- org.codehaus.jackson.type,
- org.codehaus.jackson.util,
- org.openhab.core.autoupdate,
- org.openhab.core.binding,
- org.openhab.core.events;uses:="org.openhab.core.items,org.osgi.service.event,org.openhab.core.types",
- org.openhab.core.items;uses:="org.openhab.core.types,org.openhab.core.events",
- org.openhab.core.library.items,
- org.openhab.core.library.types,
- org.openhab.core.persistence,
- org.openhab.core.scriptengine.action,
- org.openhab.core.service,
- org.openhab.core.transform,
- org.openhab.core.transform.actions,
- org.openhab.core.types,
- org.openhab.io.console,
- org.openhab.io.multimedia.tts,
- org.openhab.io.net.actions,
- org.openhab.io.net.exec,
- org.openhab.io.net.http,
- org.openhab.library.tel.items,
- org.openhab.library.tel.types,
- org.openhab.model.item.binding,
- org.openhab.model.sitemap,
- org.openhab.ui.chart,
- org.openhab.ui.items
-Bundle-ClassPath: .,
- lib/jackson-core-asl-1.9.2.jar,
- lib/jackson-mapper-asl-1.9.2.jar
-Service-Component: OSGI-INF/*.xml
+Manifest-Version: 1.0
+Bundle-Activator: org.openhab.core.compat1x.internal.CompatibilityActiva
+ tor
+Bundle-ClassPath: .,lib/jackson-core-asl-1.9.2.jar,lib/jackson-mapper-as
+ l-1.9.2.jar
+Bundle-ManifestVersion: 2
+Bundle-Name: openHAB 1.x Compatibility Layer
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.core.compat1x
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Export-Package: 
+ org.codehaus.jackson,
+ org.codehaus.jackson.annotate,
+ org.codehaus.jackson.format,
+ org.codehaus.jackson.impl,
+ org.codehaus.jackson.io,
+ org.codehaus.jackson.map,
+ org.codehaus.jackson.map.annotate,
+ org.codehaus.jackson.map.deser,
+ org.codehaus.jackson.map.deser.impl,
+ org.codehaus.jackson.map.deser.std,
+ org.codehaus.jackson.map.exc,
+ org.codehaus.jackson.map.ext,
+ org.codehaus.jackson.map.introspect,
+ org.codehaus.jackson.map.jsontype,
+ org.codehaus.jackson.map.jsontype.impl,
+ org.codehaus.jackson.map.module,
+ org.codehaus.jackson.map.ser,
+ org.codehaus.jackson.map.ser.impl,
+ org.codehaus.jackson.map.ser.std,
+ org.codehaus.jackson.map.type,
+ org.codehaus.jackson.map.util,
+ org.codehaus.jackson.node,
+ org.codehaus.jackson.schema,
+ org.codehaus.jackson.sym,
+ org.codehaus.jackson.type,
+ org.codehaus.jackson.util,
+ org.openhab.core.autoupdate,
+ org.openhab.core.binding,
+ org.openhab.core.events;uses:="org.openhab.core.items,org.osgi.service.
+ event,org.openhab.core.types",
+ org.openhab.core.items;uses:="org.openhab.core.types,org.openhab.core.e
+ vents",

```

Comment:
This looks broken is this really the better way

Reply:
see global comment (max. line width)